app/views/organizations/home/index.html.erb

```diff
@@ -11,13 +11,11 @@
           </div>
           <div class="d-flex justify-content-center">
             <% if user_signed_in? %>
-
-              <%= link_to "#{t("general.browse")} #{t("general.dogs")}",
-              adoptable_pets_path(species: "dog"),
-              class: "btn btn-primary btn-lg rounded-1 ms-2" %>
-              <%= link_to "#{t("general.browse")} #{t("general.cats")}",
-              adoptable_pets_path(species: "cat"),
-              class: "btn btn-primary btn-lg rounded-1 ms-2" %>
+              <% @unique_species.each do |species| %>
```

Comment:
This would be good to add to a partial that can be re-used across the app. I think it would make sense for it to live in `app/views/organizations/shared`